lib/blocks.php

```diff
@@ -128,3 +130,12 @@ function do_blocks( $content ) {
 	return $new_content;
 }
 add_filter( 'the_content', 'do_blocks', 10 ); // BEFORE do_shortcode().
+
+/**
+ * Loads the server-side rendering of blocks. If your block supports
+ * server-side rendering, add it here.
+ */
+function gutenberg_load_blocks_server_side_rendering() {
+	require_once GUTENBERG__BLOCKS_LIBRARY_DIR . '/latest-posts/index.php';
+}
+add_action( 'init', 'gutenberg_load_blocks_server_side_rendering' );
```

Comment:
I think there are a few other issues here, not least that `npm run package-plugin` is currently broken because this file was not added to the plugin zip package.  I'll open a PR to try to fix.